tests/apple/app_store_connect/versioning/test_review_submissions.py

```diff
@@ -27,3 +29,26 @@ def test_read(self):
         assert isinstance(review_submission, ReviewSubmission)
         assert review_submission.type is ResourceType.REVIEW_SUBMISSIONS
         assert review_submission.id == review_submission_id
+
+    def test_list(self):
+        app_id = ResourceId('1481211155')  # Banaan
+        review_submissions = self.api_client.review_submissions.list(
+            app_id,
+            Platform.IOS,
+        )
+        assert isinstance(review_submissions, List)
+        for submission in review_submissions:
+            assert isinstance(submission, ReviewSubmission)
+            assert submission.type is ResourceType.REVIEW_SUBMISSIONS
+
+
+@pytest.mark.parametrize(
+    'python_field_name, apple_filter_name', [
+        ('app', 'app'),
+        ('state', 'state'),
+        ('platform', 'platform'),
+    ],
+)
+def test_review_submissions_filter(python_field_name, apple_filter_name):
+    get_apple_filter_name = ReviewSubmissions.Filter._get_field_name
+    assert get_apple_filter_name(python_field_name) == apple_filter_name
```

Comment:
I think it makes more sense to test the public methods of this `Filter` instead of mangling with the internals.
```suggestion
def test_review_submissions_filter():
    review_submissions_filter = ReviewSubmissions.Filter(
        app=ResourceId('1481211155'),
        platform=Platform.IOS,
        state=(ReviewSubmissionState.IN_REVIEW, ReviewSubmissionState.CANCELING),
    )
    assert review_submissions_filter.as_query_params() == {
        'filter[app]': '1481211155',
        'filter[platform]': 'IOS',
        'filter[state]': 'IN_REVIEW,CANCELING',
    }

```